docs/index.rst

```diff
@@ -18,13 +18,13 @@ customer datacenters.
 .. _Scality: https://www.scality.com
 .. _Zenko: https://www.zenko.io
 
-It is based on the Kubespray_ project to reliably install a base Kubernetes
+MetalK8s, based on the Kubespray_ project, reliably installs a base Kubernetes
 cluster, including all dependencies (like etcd_), using the Ansible_
-provisioning tool. This installation is further augmented with operational
-tools for monitoring and metering, including Prometheus_, Grafana_,
-ElasticSearch_ and Kibana_. Furthermore, an "ingress controller" is deployed
-by default, based on Nginx_. All of these are managed as Helm_ packages. See
-:doc:`architecture/cluster-services` for a whole listing.
+provisioning tool. MetalK8s augments this installation with operational
+monitoring and metering tools, including Prometheus_, Grafana_, ElasticSearch_
+and Kibana_. Furthermore, MetalK8s deploys an "ingress controller" based on 
+Nginx_ by default. All of these are managed as Helm_ packages by MetalK8s. See
```

Comment:
Don't see the use for this "by MetalK8s". The name is coined too many times already in this paragraph. Furthermore, MetalK8s doesn't really manage those packages at all.

Reply:
1) Don't worry about the repetition. It's better not to be dull, but I'll pick "dull" over "vague" every time. 
2) The original sentence, iirc, was awash in passive constructions, which is why I injected a subject (apparently incorrectly). The passive voice enables us to elide the subject, and that makes it hard for an uninformed reader to find out, what's going on in the product. There's no hard-and-fast rule about this, but if the agent/subject looks clear, I'll add it. I did check with another local engineer before making that change, but it's clearly a wild pitch. 

If not MetalK8s, which component deploys the ingress controller? Does MetalK8s trigger that component's deployment of the ingress controller?